lib/proto/tupelo_rpc.proto

```diff
@@ -36,7 +36,8 @@ message StorageAdapterConfigForBadger {
 
 message StorageAdapterConfigForIpld {
   string path = 1;
-  bool offline = 2;
+  string address = 2;
```

Comment:
Is it safe here to change the field index? It could well be for all I know, I'm just asking because you're in general not supposed to change field indexes in protobuf, in order to maintain backwards compatibility.

Reply:
That is true, and something to be mindful of in the future - for now though, its just me using this